Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -672,24 +672,32 @@ public void getCurrentPosition(String callback) {
     public void isMockLocation(String callback) {
         Activity activity = bridgeComponents.getActivity();
         if (!isLocationPermissionEnabled()) return;
-        if (client != null && activity != null)
+        if (client != null && activity != null) {
             client.getLastLocation()
                     .addOnSuccessListener(activity, location -> {
-                        boolean isMock;
+                        boolean isMock = false;
```

Comment:
in java, default value of a boolean is false, no need to explicitly mention it as `false`

Reply:
the isMock variable was being assigned within an if (location != null) block that is why need to initialise the variable with a default value.

Reply:
this is not explanatory enough, still didn't get why is this required